tests/e2e/run-scanner-v4.bats

```diff
@@ -29,6 +29,17 @@ init() {
     require_environment "ORCHESTRATOR_FLAVOR"
 }
 
+export TEST_SUITE_ABORTED="false"
```

Comment:
nit: How about something like `ABORT_ON_FAILURE` which would cause teardown to abort if the last test fail, and keep all the data on the cluster?

Reply:
like the idea, allows us to get rid of the special aborting function. Thanks!